var/spack/repos/builtin/packages/pacparser/package.py

```diff
@@ -9,14 +9,22 @@
 class Pacparser(MakefilePackage):
     """pacparser is a library to parse proxy auto-config (PAC) files."""
 
+    maintainers = ['iarspider']
+
     homepage = "https://pacparser.github.io/"
-    url      = "https://github.com/manugarg/pacparser/releases/download/1.3.7/pacparser-1.3.7.tar.gz"
+    url      = "https://github.com/manugarg/pacparser/releases/download/v1.3.8/pacparser-1.3.8.tar.gz"
+    git      = "https://github.com/manugarg/pacparser.git"
 
-    version('1.3.7', sha256='eb48ec2fc202d12a4b882133048c7590329849f32c2285bc4dbe418f29aad249')
+    version('1.3.9', tag='v1.3.9')
```

Comment:
Can we use the corresponding commit sha?

Reply:
Ok.